CHANGELOG.md

```diff
@@ -7,9 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## Added
+
+- add `max_retries` and `num_threads` parameters to `rg.log` to run data logging request concurrently with backoff retry policy. See [#2458](https://github.com/argilla-io/argilla/issues/2458) and [#2533](https://github.com/argilla-io/argilla/issues/2533)
+
+## Changed
 
 - Argilla quickstart image dependencies are externalized into `quickstart.requirements.txt`. See [#2666](https://github.com/argilla-io/argilla/pull/2666)
+- The `rg.log_async` is deprecated and will be removed in next minor release.
+- The `rg.log` will compute all batches and raise an error for all failed batches.
 
```

Comment:
Perhaps mention that the batch size is now 100 under "Changed"? May be too small of a change to mention, though.